### ISSUE TITLE
trash

### ISSUE THREAD

**Issue by ravenleeblack** [MicrosoftDocs/cpp-docs#4276]
[Enter feedback here]

Like always your website is vague and doesn't offer any clear details. IT really should not be so damn difficult to make a cmake build. Starting to hate MS.
---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 04407792-a1b1-4238-16c0-2627f0aef2c9
* Version Independent ID: 04407792-a1b1-4238-16c0-2627f0aef2c9
* Content: [Walkthrough: Build and Debug C++ with Microsoft Windows Subsystem for Linux 2 (WSL 2) and Visual Studio 2022](https://learn.microsoft.com/en-us/cpp/build/walkthrough-build-debug-wsl2?view=msvc-170)
* Content Source: [docs/build/walkthrough-build-debug-wsl2.md](https://github.com/MicrosoftDocs/cpp-docs/blob/main/docs/build/walkthrough-build-debug-wsl2.md)
* Product: **visual-cpp**
* Technology: **cpp-tools**
* GitHub Login: @TylerMSFT
* Microsoft Alias: **twhitney**

**Comment by TylerMSFT** [comment]
What hung you up? Some details about what you found difficult would be helpful for me to fix this.

**Comment by ravenleeblack** [comment]
All of the actual build steps lacked info. Every time I clicked on the
build tab in menu, to set up,  it popped up a file with a bunch of text. I
assume it is a script, with no explanation from the doc, on what im suppose
to do in this script. Am I suppose to remove the builds I am not using? I'm
I supposed to copy and paste the data somewhere else. I only use C so any
doc that has a script or another language handling needs some info for it.
This really erks me. Most of microsofts documentation is poorly written and
vague like this. Then they wonder why people actually do move to linux.
This same thing with the nasm tuterial uses scripts and vague info. Then
wonder why we use VS as a pretty editor and use other tools less
complicated and more detailed. Because I couldn't figure out masm for vs, I
moved to nasm, since I couldn't figure out how to get nasm working with
yalls tuterials. I started using nasm in Ubuntu just like I have been
making Linux projects in Ubuntu alone. Thought maybe I could try again but
whatever. Everything that new ceo of yalls does sucks.
